core/src/epicli/data/common/ansible/playbooks/roles/applications/tasks/applications/rabbitmq/set-erlang-cookie.yml

```diff
@@ -1,67 +1,57 @@
 ---
 - name: Set default value for user defined cookie variable
   set_fact:
-    is_cookie_user_defined: False
-  changed_when: false
+    is_cookie_user_defined: false
 
 - name: Set cookie secret name
   set_fact:
     cookie_secret_name: "{{ rabbitmq_service_name }}-cookie"
-  changed_when: false
 
 - name: Check if cookie is user defined
   set_fact:
-    is_cookie_user_defined: True
+    is_cookie_user_defined: true
   when:
     - data.rabbitmq is defined
     - data.rabbitmq.cluster is defined
     - data.rabbitmq.cluster.cookie is defined
-  changed_when: false
 
 - name: Set cookie variable if user defined
   set_fact:
     erlang_cookie: "{{ data.rabbitmq.cluster.cookie }}"
-  when: is_cookie_user_defined == True
-  changed_when: false
+  when: is_cookie_user_defined is sameas true
```

Comment:
is it the same as `when: is_cookie_user_defined | bool`

Reply:
No it's quite different really (https://jinja.palletsprojects.com/en/2.10.x/templates/#sameas).
In this particular case using `bool` function here does not make much sense.
I've simplified all this code in the latest commit. :+1: